tasks/run-zap/README.md

```diff
@@ -4,11 +4,8 @@ Concourse task to execute [OWASP ZAP](https://www.owasp.org/index.php/OWASP_Zed_
 
 ## Usage
 
-This assumes a [Concourse](http://concourse.ci/) target named `lite`.
+This assumes a [Concourse](http://concourse.ci/) target named `lite`. Run the following from the top level of this repository:
 
-1. Run the [`filter-project-data`](../filter-project-data/) task.
-1. Run the following from the top level of this repository:
-
-    ```bash
-    fly execute -t lite -c tasks/run-zap/task.yml -i filtered-project-data=out -i scripts=.
-    ```
+```bash
+fly execute -t lite -c tasks/run-zap/task.yml -i project-data=out -i scripts=.
```

Comment:
You removed the `mkdir -p out` from a README.md (see above). You'll need it somewhere else now. Probably here.